### DISCUSSION TITLE
milvus 有向量 count 功能了吗？

### DISCUSSION THREAD

**Discussion by ponponon** [milvus-io/milvus#26801] [Q&A and General discussion]
比如我添加一个标量 user_id

我要统计集合中，user_id=1 的向量一共有多少个？有什么办法可以获取这个 count？

> 总不能用 query 把 user_id=1 的记录都读取到本地，在本地统计 count 吧？

**Comment by xiaofan-luan** [comment]
> 比如我添加一个标量 user_id
> 
> 我要统计集合中，user_id=1 的向量一共有多少个？有什么办法可以获取这个 count？
> 
> > 总不能用 query 把 user_id=1 的记录都读取到本地，在本地统计 count 吧？

Milvus 已经支持了count , see https://milvus.io/docs/query.md#Count-entities
